app/models/models.py

```diff
@@ -1,27 +1,90 @@
 """import datetime to help you get the current date and time."""
 from datetime import datetime
 from werkzeug.security import generate_password_hash
+import psycopg2
+from flask import current_app
+
+users = []
```

Comment:
These lists need to be removed since you will no longer need them; you will be persisting data into your database from now henceforth.